templates/header.redhat.html

```diff
@@ -3,31 +3,13 @@
 {% block distribution_instruction %}
 
   <p>
-  To use this repository, run the following command:
-
-  <pre class="text-white bg-dark">
-
-  sudo wget -O /etc/yum.repos.d/{{artifactName}}.repo {{ web_url }}/{{artifactName}}.repo
-  sudo rpm --import {{ web_url }}/{{organization}}-2023.key
-  </pre>
+  To install Jenkins on Fedora, use the instructions provided in our <a href="https://www.jenkins.io/doc/book/installing/linux/#fedora">Fedora installation guide</a>.
```

Comment:
End the paragraph tag as is done for the next paragraph
```suggestion
  To install Jenkins on Fedora, use the instructions provided in our <a href="https://www.jenkins.io/doc/book/installing/linux/#fedora">Fedora installation guide</a>.
  </p>
```